Yatima/Const.lean

```diff
@@ -23,61 +23,61 @@ structure Theorem where
   name: Name
   lvls: List Name
   type: Expr
-  expr: Expr
+  value: Expr
 
 structure TheoremAnon where
   lvls: Nat
   type: ExprAnonCid
-  expr: ExprAnonCid
+  value: ExprAnonCid
 
 structure TheoremMeta where
   name: Name
   lvls: List Name
   type: ExprMetaCid
-  expr: ExprMetaCid
+  value: ExprMetaCid
 
 structure Opaque where
   name: Name
   lvls: List Name
   type: Expr
-  expr: Expr
+  value: Expr
   safe: Bool
 
 structure OpaqueAnon where
   lvls: Nat
   type: ExprAnonCid
-  expr: ExprAnonCid
+  value: ExprAnonCid
   safe: Bool
 
 structure OpaqueMeta where
   name: Name
   lvls: List Name
   type: ExprMetaCid
-  expr: ExprMetaCid
+  value: ExprMetaCid
 
 inductive DefSafety where
-| _safe
-| _unsafe
-| _partial
+| safe
+| «unsafe»
```

Comment:
I don't like «» raw identifiers, they're not ergonomic. Prefer `_` prefixing instead

Reply:
Answered by @arthurpaulino out of band:
> We don't need unicode actually. We only need unicode to declare them because they're standalone words. When we actually use them, we're going to do like Const.theorem and it's going to work just fine

Which is fine by me then